src/avfoundation.cs

```diff
@@ -9160,6 +9196,23 @@ interface AVCapturePhotoSettings : NSCopying
 		[NoWatch, NoTV, NoMac, iOS (12, 0)]
 		[Export ("embedsPortraitEffectsMatteInPhoto")]
 		bool EmbedsPortraitEffectsMatteInPhoto { get; set; }
+
+		[Internal]
+		[NoWatch, NoTV, NoMac, iOS (12, 0)]
+		[Export ("availableRawEmbeddedThumbnailPhotoCodecTypes")]
+		NSString[] _AvailableRawEmbeddedThumbnailPhotoCodecTypes { get; }
```

Comment:
You might be able to just use `[BindAs (typeof (AVVideoCodecType []))]` here instead of the Wrap

Reply:
API will stay the same, right? It is a nice to have, so I'll prefer to reduce the number of changes :) I'll use BindAs the next time.

Reply:
Kind of, we will actually generate the strong API with the export so not really the same but it is the same for the end user perspective unless you subclass heh

Reply:
@mandel-macaque ^ might was well pick that fix up